errors/errors.go

```diff
@@ -56,6 +56,9 @@ type Error struct {
 
 	// Tag is a string identifying the error, used with HTTP responses only.
 	tag string
+
+	// Category is a string identifying the category of the error, used for error code metrics only.
```

Comment:
```suggestion
	// Category is a string identifying the category of the error, for example: ___, ___, ____, used for error code metrics only. 
```
small nit: can you expand here on types of categories to be clearer. Then, looks like you'll need to tweak this file too `errors/codes.go`. Otherwise, the changes look fine :)